resources/pool.rb

```diff
@@ -269,8 +269,12 @@ def configure
     end
 
     if new_resource.no_managed_code
-      converge_if_changed :runtime_version do
+      Chef::Log.warn("ran into no managed runtime #{new_resource.runtime_version} | #{current_resource.runtime_version}")
```

Comment:
What's your intention with #{new_resource.runtime_version} | #{current_resource.runtime_version}" here?

Reply:
I honestly don't know why i did the above, it looks like i hit a bug with Chef `converge_if_changed`